cmd/offboard/offboard.go

```diff
@@ -0,0 +1,101 @@
+package offboard
+
+import (
+	"errors"
+	"fmt"
+	"slices"
+
+	"github.com/spf13/cobra"
+
+	"github.com/open-sauced/pizza-cli/v2/pkg/config"
+)
+
+type Options struct {
+	offboardingUsers []string
+
+	// config file path
+	configPath string
+
+	// repository path
+	path string
+
+	// from global config
+	ttyDisabled bool
+}
+
+const offboardLongDesc string = `[WIP] Removes a user from the \".sauced.yaml\" config and \"CODEOWNERS\" files.
+Requires the user's name OR email.`
```

Comment:
Instead of `[WIP]`, could be "Caution: Experimental command."

Could also include `examples` like we do elsewhere